src/coreclr/tools/Common/TypeSystem/IL/Stubs/RuntimeHelpersIntrinsics.cs

```diff
@@ -75,13 +75,22 @@ public static MethodIL EmitIL(MethodDesc method)
                         result = true;
                         break;
                     default:
+                        result = false;
                         var mdType = elementType as MetadataType;
                         if (mdType != null && mdType.Name == "Rune" && mdType.Namespace == "System.Text")
```

Comment:
```suggestion
                        if (mdType != null && mdType.Module == mdType.TypeSystemContext.SystemModule && mdType.Name == "Rune" && mdType.Namespace == "System.Text")
```
Unrelated bug (we do not want to pick user types that happen to have this name)